### PR TITLE
Add inspection-batch workflow example

### PR DESCRIPTION
**Pull request by CermakM**
This workflow submits a Job which creates `n` pod replicas and runs them in a sequence.

successCondition: `n` pods have finished successfully and are present in the cluster

The workflow simulates the scenario when we want to create a batch of inspections.

Signed-off-by: Marek Cermak <macermak@redhat.com>

new file:   examples/inspection-batch.yaml

### CODE REVIEW

**Comment by CermakM** [comment]
[Create examples of Argo Workflows.](https://trello.com/c/rbSVX7o4/730-create-examples-of-argo-workflows)

File: examples/inspection-batch.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          completions: {{workflow.parameters.batch_size}}
+          template:         
+            metadata:
+              name: "{{workflow.name}}"
+              labels:
+                mark: cleanup
+            spec:
+              containers:
+              - name: fake-inspection
+                image: alpine:latest
```
Comment (goern) [inline examples/inspection-batch.yaml]: ubi8?!

**Review by goern — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Build failed.

- [thoth-coala ](https://managesf.thoth-station.ninja/logs/1/1/f6de2447a7bc563194943547b7e9113412cd1035/check/thoth-coala/5849c53/) : FAILURE in 34s

**Comment by CermakM** [comment]
recheck

**Review by goern — APPROVED** [review]

**Pull request merged**